sklearn/utils/extmath.py

```diff
@@ -330,6 +339,18 @@ def randomized_svd(M, n_components, *, n_oversamples=10, n_iter='auto',
                           type(M).__name__),
                       sparse.SparseEfficiencyWarning)
 
+    if random_state == 'warn':
+        warnings.warn(
+            "If 'random_state' is not supplied, the current default "
+            "is to use 0 as a fixed seed. This will change to  "
+            "None in version 1.2 leading to non-deterministic results "
+            "that better reflect If you want to silence this warning, "
```

Comment:
that better reflect the nature of randomized_svd. If you want...

Reply:
sure